packages/patternfly-3/patternfly-react-extensions/sass/patternfly-react-extensions/_classification-banner.scss

```diff
@@ -0,0 +1,123 @@
+$classification-banner-height: 28px;
+$classification-banner-height-top: 3px;
+$classification-banner-padding-large: 42px;
+$classification-banner-padding-sml: 20px;
+
+.classification-banner-pf-banner{
+    background: $color-pf-black;
+    background-clip: padding-box;
+    color: white;
+    height: $classification-banner-height;
+    line-height: $classification-banner-height;
+    font-size: 10px;
+    position: relative;
+}
+
+.classification-banner-pf-banner-top{
+    width: 100%;
+    height: 31px;
+}
+.classification-banner-pf-banner-top::before{
+    content: '';
+    background: rgba(255,255,255,0.5);
+    display: block;
+    width: 100%;
+    height: 3px;
+}
+
+.classification-banner-pf-banner-bottom{
+    width: 100%;
+}
+
+.classification-banner-pf-hide{
+    display: none;
+}
+
+.classification-banner-pf-banner-classified{
+    background: $color-pf-green;
+}
+
+.classification-banner-pf-banner-unclassified{
+    background: $color-pf-red;
+}
+
+.classification-banner-pf-banner-proprietary-level{
+    background: $color-pf-blue;
+}
```

Comment:
@zeyap we should define color backgrounds as they are configurable. Color values need to be provided. Because of this, I don't think that we should have `classification-banner-pf-banner-classified`, `classification-banner-pf-banner-unclassified`, and `classification-banner-pf-banner-proprietary-level`.

Reply:
I have just cancelled the default values for classification level and corresponding colors and instead added the title parameter. 
But can we maybe keep the classification-banner-pf-green, classification-banner-pf-red classes or so, and user can make use of these colors by setting bannerColor as 'pf-green' or 'pf-red' because I feel it could be useful to have access to patternfly's color schemes.